TO-DO-APP/app/src/main/java/com/example/to_do_app/DAO.kt

```diff
@@ -1,22 +0,0 @@
-package com.example.to_do_app
-
-import androidx.room.*
-
-@Dao
-interface DAO {
-    @Insert
-    suspend fun insertTask(entity: Entity)
```

Comment:
instead of passing whole entity class. Pass only required task item and use insert method for inserting entry in table. 
Mean user proper query for database entry.